docs/Developers/Git_Basics.rst

```diff
@@ -245,6 +245,12 @@ the desired branch:
     $ git fetch origin
     $ git rebase origin/master
 
+.. note::
+
+    ``origin`` is the remote of https://github.com/coala/coala
+    or https://github.com/coala/coala-bears. It is not the remote
+    of your fork.
```

Comment:
I think one source of confusion could have arisen from users who cloned from their fork. The line, "You should ideally clone the fork so that gets set to ‘origin’ automatically.", can be found right after users were told to clone from the original repo. Users who chose to follow that advice would then run into issues down the line, as our working assumption is that they cloned from the original repo, not their fork.

Personally, I prefer to clone my fork and set the original repo as `upstream`. But to be consistent throughout the guide, I think this line should be removed so users aren't told to do one thing (clone original repo), but advised to do another (clone your fork).